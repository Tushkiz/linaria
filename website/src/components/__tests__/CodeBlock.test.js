/* @flow */
import React from 'react';
import render from 'preact-render-to-string';
// import { collect } from 'linaria/server';
import CodeBlock from '../CodeBlock';
// import Container from '../Container';

test('CodeBlock', () => {
  const tree = render(<CodeBlock text="testxx" />, {}, { pretty: true });
  // const tree2 = render(
  //   <Container>
  //     <CodeBlock text="testxx" />
  //   </Container>,
  //   {},
  //   { pretty: true }
  // );
  // delete window.document;
  // console.log(Module);
  // const Module = require('module');
  // console.log(require('../../../../build/sheet').default.styles());
  // const sheet = Module._cache[require.resolve('linaria/build/sheet')];
  console.log(document.styleSheets);
  // console.log(require.resolve('linaria/build/sheet'));
  // console.log(require.requireActual('linaria/build/sheet').default.styles());
  // console.log(sheet.exports.default.styles());
  // console.log(sheet);

  // expect(tree).toMatchSnapshot();
});
