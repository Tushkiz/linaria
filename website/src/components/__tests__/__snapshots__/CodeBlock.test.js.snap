// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CodeBlock 1`] = `
"<pre class=\\"code__qxo8f1\\">
	<code></code>
</pre>"
`;
